packages/node_modules/@webex/plugin-meetings/src/meeting/state.js

```diff
@@ -71,6 +71,15 @@ const MeetingStateMachine = {
           ],
           to: MEETING_STATE_MACHINE.STATES.ENDED
         },
+        //  when ending a meeting it must be from  JOINED, or ERROR states, and transitions it to the ENDED state
+        {
+          name: MEETING_STATE_MACHINE.TRANSITIONS.END,
+          from: [
```

Comment:
The LEAVE transition is defined also for RINGING/ANSWERED/DECLINED states - I guess these are used only in 1-1 calls. I'm not sure are we allowing `endMeetingForAll()` to be called for 1-1 calls? If no, then I don't see any code catching that case and throwing appropriate error. If yes, then theses states should be added here.

Reply:
ok, thanks, maybe we should have a separate test for the case of 1-1 call, but I'm approving the PR anyway